### ISSUE TITLE
why is aria-description a global attribute without any prohibitions

### ISSUE THREAD

**Issue by stevefaulkner** [w3c/aria#1397]
Looking at the latest editors draft of ARIA 1.2 [`aria-description`](https://w3c.github.io/aria/#aria-description) is listed as a global attribute, but unlike [`aria-label`](https://w3c.github.io/aria/#aria-label) it is not prohibited on certain elements. This seems wrong. if a role cannot have a label why should it be able to have a description? Likewise for `aria-describedby`

**Comment by aleventhal** [comment]
Hi Steve, and thanks for the question. This is intentional. Descriptions are annotations, and will be useful in a variety of contexts. These are flat descriptions, and aria-details are structured descriptions. Something without a role or label, but an annotation, can be a container for other content, where that content is annotated.

For use cases and more information, please see:
 https://github.com/aleventhal/aria-annotations/blob/master/README.md#simplified-aria-annotations-proposal--explainer

**Comment by stevefaulkner** [comment]
@aleventhal i get that, my concern is that SRs in particular don't support and won't support descriptions on many elements, which is the current case with `aria-describedby`. And that if they do, developers will misuse and SR vendors will end up turning off descriptions due to user complaint. What have SR vendors signaled?

**Comment by aleventhal** [comment]
@stevefaulkner, JAWS and NVDA are actively working on it.
ChromeVox has some support but more work to do.
Talkback and VoiceOver -- no progress yet.

Here's a "brainstorm" of UX ideas we have provided for screen readers vendors; however, each screen reader vendor will vary from this per what works best for them.
https://docs.google.com/document/d/1DYrsHDk6Y9071A1fRg8g8NZ_DtGYraX5BcVY-taBzMQ/edit
